enterprise/server/remote_execution/containers/firecracker/firecracker.go

```diff
@@ -893,7 +893,8 @@ func copyStaticFiles(ctx context.Context, env environment.Env, workingDir string
 	ctx, span := tracing.StartSpan(ctx)
 	defer span.End()
 
-	locateBinariesOnce.Do(func() {
+	locateBinariesOnce, _ := locateBinariesOnceMap.LoadOrStore(workingDir, &sync.Once{})
```

Comment:
not sure I understand why the locateBinariesOnce had to be changed to a sync.Map -- are there tests that use multiple values of workingDir ? if not, my (possibly flawed) understanding was that each test should be run as its own process, so the sync.Once would be re-evaluated each time for the workingDir unique to each test

Reply:
No, `sync.Once` only runs once overall, not once per test. I initially was under the same misapprehension.